src/parser/view_transform.c

```diff
@@ -1586,6 +1587,27 @@ mq_remove_select_list_for_inline_view (PARSER_CONTEXT * parser, PT_NODE * statem
 	}
     }
 
+  /* check whether to unset hidden col. If no hidden column in original, remove hidden settings */
+  is_unset_hidden_col = true;
+  for (col = subquery->info.query.q.select.list; col; col = col->next)
+    {
+      if (col->flag.is_hidden_column)
+	{
+	  is_unset_hidden_col = false;
```

Comment:
`false`도 문제 없지만 다른 코드에서는 1 또는 0을 설정하고 있습니다.
```suggestion
	  is_unset_hidden_col = 0;
```